test/other/metadce/test_metadce_hello_O2.size

```diff
@@ -1 +1 @@
-2054
+9174
```

Comment:
This code size increase is quite concerning. From the `.funcs` above it looks like it's due to adding malloc support .

Reply:
After some changes, it looks like malloc isn't showing up in `.funcs`, but there's still a slight code size increase. Is this to be expected from the things I've added to `library_wasmfs.js`?

Reply:
A small increase with WasmFS+FORCE_FILESYSTEM is expected, but the changes here look odd. For example `test/other/test_unoptimized_code_size.js.size` doesn't even use WasmFS. Perhaps try `--rebaseline` after `emsdk install tot` and `emcc --clear-cache` to make sure you're building the latest?

Reply:
I think I've been seeing the code size increases on main as well. I did both `emsdk install tot`, `emsdk activate tot`, and `emcc --clear-cache`. Are you able to reproduce this?
```
test_unoptimized_code_size (test_other.other) ... Clearing existing test directory
  seen wasm size: 12153 (expected: 12153) (delta: 0), ratio to expected: 0.000000
  seen js size: 60352 (expected: 58664) (delta: 1688), ratio to expected: 0.028774
  seen wasm size: 11600 (expected: 11600) (delta: 0), ratio to expected: 0.000000
  seen js size: 32938 (expected: 31978) (delta: 960), ratio to expected: 0.030021
  seen wasm size: 12153 (expected: 12153) (delta: 0), ratio to expected: 0.000000
  seen js size: 59249 (expected: 57603) (delta: 1646), ratio to expected: 0.028575
ok

----------------------------------------------------------------------
Ran 1 test in 2.106s

OK
(base) PS C:\Users\james\Documents\GitHub\emscripten> git branch
  library-wasmfs-ioctl
  library-wasmfs-mkdev
  library-wasmfs-mmap
  library-wasmfs-mmap-august
  library-wasmfs-mount
  library-wasmfs-mount-acct
  library-wasmfs-readlink-fix
  library-wasmfs-syncfs
* main
```